src/vm/dac/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
-
 include(${CLR_DIR}/dac.cmake)
+include(CMakeDetermineCXXCompiler)
```

Comment:
I am not sure how the CMAKE_CXX_COMPILER_VERSION can be empty here. When I run both native and cross build with both clang and gcc5, the variable is always set correctly here (added a message to print it).

Reply:
What I have observed is that there are multiple passes to this file, and one of those has missing value without this include, and thus fails the gcc 8 build with the error in description. Maybe if you add an `else()` case for debug message, that will hit as well.

Reply:
I was just printing the content of the variable unconditionally like:
```cmake
message("CMAKE_CXX_COMPILER_VERSION is ${CMAKE_CXX_COMPILER_VERSION}")
```
It was printed just once during a clean build and it was set.

Reply:
You are right. I cannot reproduce the error either without the `include` statement with clean build (before I was hitting it constantly without this statement in the same system). Thanks!